infra/terraform/environments/dev/main.tf

```diff
@@ -57,6 +57,19 @@ module "user_nfs" {
     performance_mode = "maxIO"
 }
 
+module "user_nfs_softnas" {
+    source = "../../modules/user_nfs_softnas"
+
+    env = "${var.env}"
```

Comment:
I know the default for the variable `softnas_role_name` is `SoftNAS_HA_IAM` so shouldn't matter but I wonder if we should pass it in as `"${aws_iam_role.softnas}"`. Also, any reason to not create this role/policy inside the module (as far as I can't tell it's not used anywhere else outside it).